xds/internal/balancer/lrs/balancer.go

```diff
@@ -219,34 +227,21 @@ type xdsClientWrapper struct {
 	loadWrapper *loadStoreWrapper
 }
 
-func newXDSClientWrapper() *xdsClientWrapper {
+func newXDSClientWrapper(c xdsClientInterface) *xdsClientWrapper {
```

Comment:
Same here. Do we need this wrapper type?

Reply:
See reply in the eds balancer.